sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -1359,6 +1404,7 @@ lsc_atomic_update(AccessorTy acc, __ESIMD_NS::simd<uint32_t, N> offsets,
                                 _DS, _VS, _Transposed, N>(
           pred.data(), offsets.data(), src0.data(), src1.data(), si);
   return detail::lsc_format_ret<T>(Tmp);
+#endif
```

Comment:
please add some tests

Reply:
Added some compilation tests. Tested ESIMD tests from llvm-test-suite with -sycl-esimd-force-stateless-mem switch.
These tests would fail with forced stateless memory accesses because the intrinsics they use do not have USM equivalents:
  SYCL :: ESIMD/histogram.cpp
  SYCL :: ESIMD/histogram_2d.cpp
  SYCL :: ESIMD/histogram_raw_send.cpp
  SYCL :: ESIMD/linear/linear.cpp
  SYCL :: ESIMD/mandelbrot/mandelbrot.cpp
  SYCL :: ESIMD/mandelbrot/mandelbrot_spec.cpp
  SYCL :: ESIMD/vadd_2d.cpp
  SYCL :: ESIMD/vadd_raw_send.cpp